.ci/set_dev_dependencies.jl

```diff
@@ -97,6 +98,6 @@ function set_dev_dependencies(
 end
 
 if abspath(PROGRAM_FILE) == @__FILE__
-    deps = get_filtered_dependencies(r"^QED*")
+    deps = get_filtered_dependencies(r"^QuantumElectrodynamics*")
```

Comment:
I am unsure if this is right because no package passes the filter `QuantumElectrodynamics*`except `QuantumElectrodynamics.jl` itself. 

I think `QED*` is right here.

Reply:
I think you're right, but I'd like to ask @SimeonEhrig too because I'm not sure if maybe all the packages need to match here, i.e., maybe we need `^(QED|QuantumElectrodynamics)*` now?

Reply:
`^(QED|QuantumElectrodynamics)*` looks correct. I have to verify it, but the regex needs to include at least `QED` and `QuantumElectrodynamics`.